src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img src="Rectangle 4.png"
+      class="card__img"
+      alt="picture"
+      >
+      <p class="card__text">
+        APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="text text--one">Product code: 195434</p>
+
+      <div class="card__stars-and-reviews">
+        <div class="card__stars">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <p class="text text--two">Reviews: 5</p>
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/97547824/2f5cc73a-42ee-4e6e-b9b5-3f330ef2faf4)
```suggestion
        <p class="card__reviews">Reviews: 5</p>
```
or 
```suggestion
        <p class="card__text card__text--reviews">Reviews: 5</p>
```